python-package/lightgbm/basic.py

```diff
@@ -1853,6 +1854,122 @@ def free_network(self):
         self.network = False
         return self
 
+    def trees_to_dataframe(self):
+        """Parse the fitted model and return in an easy-to-read pandas DataFrame.
+
+        Returns
+        -------
+        result: pandas DataFrame
```

Comment:
Please fix formatting issue [I mentioned in the previous review](https://github.com/microsoft/LightGBM/pull/2592#discussion_r364430743):
```suggestion
        result : pandas DataFrame
```